drivers/stepper/Kconfig.step_dir

```diff
@@ -0,0 +1,7 @@
+# Copyright 2024 Fabian Blatz
```

Comment:
Let's add SPDX - Filecopyright text thingy here.

Reply:
Added it :^)